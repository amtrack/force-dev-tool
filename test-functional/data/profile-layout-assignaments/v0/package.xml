<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
  <types>
    <members>Account.googleButton</members>
    <name>WebLink</name>
  </types>
  <types>
    <members>Account-Account %28Marketing%29 Layout</members>
    <members>Account-Account %28Sales%29 Layout</members>
    <members>Account-Account %28Support%29 Layout</members>
    <members>Account-Account Layout</members>
    <name>Layout</name>
  </types>
  <types>
    <members>Account.Customer</members>
    <members>Account.Person</members>
    <name>RecordType</name>
  </types>
  <types>
    <members>Admin</members>
    <name>Profile</name>
  </types>
  <version>46.0</version>
</Package>
