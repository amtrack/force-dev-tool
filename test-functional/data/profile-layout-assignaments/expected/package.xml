<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>Admin</members>
        <name>Profile</name>
    </types>
    <version>41.0</version>
</Package>
